app/Domains/Accounts/Actions/CreateAccount.php

```diff
@@ -32,7 +32,10 @@ public function handle(array $params): Account
     public function rules(): array
     {
         return [
-
+            'name' => ['required', 'string', 'max:255'],
+            'balance' => ['required', 'integer', 'min:0'],
+            'type' => ['required', 'in:' . implode(',', [AccountTypesEnum::CASH, AccountTypesEnum::BANK, AccountTypesEnum::MOBILE])],
```

Comment:
1. AccountTypesEnum has to be injected into the file before it can used.
2. Also, use `AccountTypesEnum::asArray()` instead, this would be a more dynamic approach.
If we make changes to `AccountTypesEnum` in the future, we'll also have to change this file with this approach.